lib/ClangImporter/ImportDecl.cpp

```diff
@@ -345,6 +345,23 @@ static bool isNSDictionaryMethod(const clang::ObjCMethodDecl *MD,
   return true;
 }
 
+static void forEachDistinctName(
```

Comment:
Could you provide a comment on what "distinct" means and any assumptions? Swift name is distinct for Clang name? Maybe each name for each version for the imported decl? Or can a given version have many Swift names?